client/src/components/Layout/Events/Events.js

```diff
@@ -21,23 +24,39 @@ class Events extends Component {
       .catch((err) => console.log(err))
   }
 
+  setSearchQuery = name => this.setState({ searchQuery: name.target.value})
+  
+
   render () {
-    const { events } = this.state
+  const { events, searchQuery} = this.state
+  const filterSearch = events.filter(event => event.event_title.toLowerCase().indexOf(searchQuery.toLowerCase())!== -1)
     return (
+
       <div>
         <NavBar />
+        <div className='SearchEvent'>
+          <div>
+            <input type='text'
+              placeholder='Search for Event ... '
+              onChange={this.setSearchQuery}
+              value ={searchQuery}
+            />
+          </div>
+          <div className='Search'>
+            <img
```

Comment:
If you decide to keep the icon, then its size and position should be more friendly